Libraries/src/Amazon.Lambda.KafkaEvents/KafkaEvent.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace Amazon.Lambda.KafkaEvents
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Apache Kafka event
+    /// https://docs.aws.amazon.com/lambda/latest/dg/with-msk.html
+    /// https://docs.aws.amazon.com/lambda/latest/dg/with-kafka.html
+    /// </summary>
+    public class KafkaEvent
+    {
+        /// <summary>
+        /// The source of the event.
+        /// </summary>
+        public string EventSource { get; set; }
+
+        /// <summary>
+        /// The ARN of the event source.
+        /// </summary>
+        public string EventSourceARN { get; set; }
+
+        /// <summary>
+        /// Initial list of brokers as a CSV list of broker host or host:port.
+        /// </summary>
+        public string BootstrapServers { get; set; }
+
+        /// <summary>
+        /// List of Kafka event records.
+        /// </summary>
+        public IDictionary<string, IList<KafkaEventRecord>> Records { get; set; }
+
+        /// <summary>
+        /// Kafka event record.
+        /// </summary>
+        public class KafkaEventRecord
+        {
+            /// <summary>
+            /// The topic associated with the event record.
+            /// </summary>
+            public string Topic { get; set; }
+
+            /// <summary>
+            /// The partition associated with the event record.
+            /// </summary>
+            public int Partition { get; set; }
```

Comment:
partition is defined as string property as per https://docs.aws.amazon.com/lambda/latest/dg/with-msk.html

de-serialization failed for me when using the lambda test tool. It should be string.

That said, could you also add sample request for local debugging?

There are some example requests [here](https://github.com/aws/aws-lambda-dotnet/tree/087590ce99274e16e26d37e1dfd73b0b71d1230a/Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Resources/SampleRequests)

Reply:
Done. I added kafka event request to samples.